stable/rbac-manager/values.yaml

```
@@ -23,3 +23,13 @@ affinity: {}
 
 podAnnotations: {}
 podLabels: {}
+
+service:
+  enabled: false
+  annotations: {}
+
+serviceMonitor:
+  enabled: false
+  annotations: ""
```

I know it doesn't change anything technically, but this will indicate to the end-user that this is a map instead of a string.

```suggestion
  annotations: {}
```